R/pkg/R/DataFrame.R

```diff
@@ -1069,6 +1079,8 @@ setMethod("first",
 #'
 #' @param x A SparkDataFrame
 #'
+#' @family SparkDataFrame functions
+#' @rdname tordd
```

Comment:
@felixcheung ok, should we also remove these two lines in `toJSON` part in line 631?